### PR TITLE
Add rbs-rules scm function

### PR DESCRIPTION
**Pull request by AmeBel**
rbs-rules returns the alias for rules in a rule-base.

### CODE REVIEW

**Comment by AmeBel** [comment]
@misgeatgit @williampma pls review, thanks

File: opencog/rule-engine/InferenceSCM.cc [hunk 1]
```diff
@@ -131,6 +134,32 @@ Handle do_backward_chaining(AtomSpace* as,
 #endif
 }
 
+HandleSeq get_rulebase_rules(AtomSpace* as,
```
Comment (misgeatgit) [inline opencog/rule-engine/InferenceSCM.cc]: May be you should move this to UREConfigReader class.

File: opencog/rule-engine/InferenceSCM.cc [hunk 2]
```diff
@@ -131,6 +134,32 @@ Handle do_backward_chaining(AtomSpace* as,
...
+                          const Handle& rbs)
+{
+#ifdef HAVE_GUILE
+    if (Handle::UNDEFINED == rbs)
+        throw RuntimeException(TRACE_INFO,
+            "InferenceSCM::get_rulebase_rules - invalid rulebase!");
+
+	UREConfigReader ure_config(*as, rbs);
```
Comment (misgeatgit) [inline opencog/rule-engine/InferenceSCM.cc]: Indentation.

File: opencog/rule-engine/InferenceSCM.cc (hunk 1 above)
Comment (AmeBel) [inline opencog/rule-engine/InferenceSCM.cc]: there is a get_rules already, you mean creating one that returns a `HandleSeq`?

**Comment by misgeatgit** [comment]
@AmeBel  Whats the purpose these commits BTW? What is it solving?

**Comment by AmeBel** [comment]
In OpenPsi an active-scheme-pool is a rulebase. Unlike the r2l or pln the rulebase isn't static it is dynamic,i.e, the set of rules change when goals changes. 

Yes, this searching can be done using scheme, but then should the definition of a rule-pattern change then I have to update scheme-function.

File: opencog/rule-engine/InferenceSCM.cc (hunk 1 above)
Comment (misgeatgit) [inline opencog/rule-engine/InferenceSCM.cc]: OK, I am wrong. Looks it is in the right place.

**Comment by misgeatgit** [comment]
Thanks, now I understood.

**Pull request merged**